Gemfile

```diff
@@ -20,40 +20,34 @@ gem "sass-rails", "~> 6.0"
 gem "turbolinks", "~> 5"
 gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
 gem "uglifier", ">= 1.3.0"
+gem "dotenv-rails"
 
 group :development do
   gem "listen", ">= 3.0.5", "< 3.2"
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
   gem "web-console", ">= 3.3.0"
+  gem "better_errors"
+  gem "html2haml"
```

Comment:
There are gems in here that don't seem to be related to the PR..?

Reply:
Good point, I had a bit of a re-org of the gemfile and it ended up in here, I think I'll yank it